### PR TITLE
Create an `AnalysisPromptCategory` entity

### PR DESCRIPTION
**Pull request by isxoqovxasanboy**

### CODE REVIEW

**Review by rashidovasadbek — DISMISSED** [review]

**Review by botirjonabdumannonovv — APPROVED** [review]

**Review by isxoqovxasanboy — COMMENTED** [review]
close #88

**Review by rashidovasadbek — DISMISSED** [review]

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Domain/Extensions/EnumExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel;
+
+namespace Feedback.Analyzer.Domain.Extensions;
+
+public static class EnumExtensions
+{
+    public static string GetDisplayName(this Enum value)
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Domain/Extensions/EnumExtensions.cs:7]: Add docs

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/EntityConfiguration/AnalysisPromptCategoryConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+    public void Configure(EntityTypeBuilder<AnalysisPromptCategory> builder)
+    {
+        builder
+            .Property(category => category.Category)
+            .HasConversion<string>()
+            .IsRequired();
+
+        builder
+            .HasIndex(category => category.Category);
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/EntityConfiguration/AnalysisPromptCategoryConfiguration.cs:17]: Change to unique index 


Implementation : 

```csharp
builder
         .HasIndex(category => category.Category)
         .IsUnique();
```

File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/EntityConfiguration/AnalysisPromptCategoryConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace Feedback.Analyzer.Persistence.EntityConfiguration;
+
+public class AnalysisPromptCategoryConfiguration : IEntityTypeConfiguration<AnalysisPromptCategory>
+{
+    public void Configure(EntityTypeBuilder<AnalysisPromptCategory> builder)
+    {
+        builder
+            .Property(category => category.Category)
+            .HasConversion<string>()
```
Comment (sulton-max) [inline src/Feedback.Analyzer.Backend/Feedback.Analyzer.Persistence/EntityConfiguration/AnalysisPromptCategoryConfiguration.cs:13]: Add length constraint for 128 chars

**Review by sulton-max — CHANGES_REQUESTED** [review]

**Review by botirjonabdumannonovv — APPROVED** [review]

**Review by rashidovasadbek — APPROVED** [review]

**Review by sulton-max — APPROVED** [review]
Good job

**Pull request merged**